cmd/lokomotive-controller/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+	controller "github.com/kinvolk/lokomotive/pkg/controller"
```

Comment:
nit: A space between these two imports to highlight vendor vs our own imports.

Reply:
space => newline in this case :)